### DISCUSSION TITLE
Can you get a parameter value from standard in if it was not specified ?

### DISCUSSION THREAD

**Discussion by federico-paolillo** [Tyrrrz/CliFx#112] [Q&A] [answered]
I'm looking for a way to support shell pipes in my cli application in order to do something like: `echo "a b c" | mycli`
Of course I would still like to use the cli application as normal, like: `mycli a b c` and get the parameter help text.

Would this be possible ?

As far as I could see CliFx explicitly makes all the parameters required, therefore there is no way to do that using a parameter.

**Comment by Tyrrrz** [comment, marked as answer]
No, this is not possible.
